core.coffee

```diff
@@ -434,6 +447,35 @@
 						url += '?' + queryOptions.join('&')
 					return url
 
+			compileBatch: (data) ->
+				if !utils.isArray(data)
+					throw new Error('Batch must be passed an array')
+				data = utils.cloneDeep(data)
```

Comment:
Should we implement our own version of these functions inside `core.coffee` or add the requirements for `utils`? Same applies for `omit`